feature/about/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_what_is_droidkaigi">What is DroidKaigi?</string>
+    <string name="about_description">DroidKaigiはエンジニアが主役のAndroidカンファレンスです。 Android技術情報の共有とコミュニケーションを目的に、2020年2月20日(木)、21日(金)の2日間開催します。</string>
```

Comment:
Thanks!
From https://droidkaigi.jp/2020/en/
`DroidKaigi is a conference tailored for Android developers. It's scheduled to take place on the 20 and 21 of February 2020.`